### ISSUE TITLE
Profiles editor is not accessible / usable by keyboard

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#223297]
Testing #223080

The profiles editor is not usable by keyboard (only by mouse).

I cannot put focus on this widget and use the arrow keys/tab to select an option:

![Image](https://github.com/user-attachments/assets/084032d2-f27f-46b7-b0da-4bf93570e925)

**Comment by sandy081** [comment]
It works with `tab` button for me, can you please double check?

**Comment by hediet** [comment]
Ah yes, it works indeed with tab, but it seems like the active button uses the focus-border as border without being focused, so you cannot see that it has focus.

**Comment by hediet** [comment]
> Ah yes, it works indeed with tab, but it seems like the active button uses the focus-border as border without being focused, so you cannot see that it has focus.

I do believe that it should be visibly distinguishable if an element has focus or not.

It is not a high priority issue, but we shouldn't forget about it, especially when this component is used in other places.

**Comment by sandy081** [comment]
This seems to be an existing UX issue with other radio buttons too, like in Search View.

@isidorn Assigning to you as a UX designer fallback
